www/docs/en/dev/cordova/storage/storage.md

```diff
@@ -162,10 +162,12 @@ IndexedDB provides a simple and easy to understand data model, much like LocalSt
 But unlike LocalStorage, you can create multiple databases, with multiple stores per
 database, and its asynchronous API and search indexes provide performance benefits.
 
-IndexedDB is supported by the underlying WebView on the following Cordova platforms:
+IndexedDB is supported by the underlying WebView on all platforms,
+with known limitations on web browser and Windows.
 
-- Windows (with some limitations)
-- Android (4.4 and above)
+### Web browser limitations
+
+The actual behavior may depend on which browser is used. There may be differences on Edge and Firefox browsers, for example.
```

Comment:
IMO we should not use Edge as an example, because it soon is Chromium based, too. Maybe just use Safari or Chrome instead?

Reply:
How about this:

```suggestion
The actual behavior may depend on which browser is used. There could be differences between the behavior on Safari and Firefox browsers, for example.
```